docs/spark/spark-procedures.md

```diff
@@ -105,7 +105,7 @@ This procedure invalidates all cached Spark plans that reference the affected ta
 
 #### Example
 
-Roll back `db.sample` to one day
+Roll back `db.sample` to one day before (assuming today is TIMESTAMP '2021-06-31 00:00:00.000')
```

Comment:
I would just change this to "Roll back to a specific day", just to keep it simple.

Reply:
yeah, I like this suggestion.